LICENSE.txt

```diff
@@ -0,0 +1,21 @@
+The MIT License
```

Comment:
@erikvold, this technically just being a fork of your old/ex-project restartless-restart-ffext I forked at some point, in case the origin would disappear :) – @oeekker seems to have taken to updating it to something working today here, and added a nice MIT license file. Is that cool with you?